OutboundCallReminder/.project

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can we have this project as a pure maven project, not eclipse project and not check-in any eclipse ide specific file? I see the other samples are created as mvn projects only as well.